src/pages/projectSettings/tabs/EventLogTab/EventLogTab.tsx

```diff
@@ -38,11 +48,108 @@ export const EventLogTab: React.FC = () => {
     },
   });
 
+  type logEntry = {
+    timestamp: Date;
+    user: string;
+    before?: RepoEventSettingsFragment | ProjectEventSettingsFragment;
+    after?: RepoEventSettingsFragment | ProjectEventSettingsFragment;
+  };
+
+  const eventData = isRepo
+    ? repoEventData?.repoEvents?.eventLogEntries
+    : projectEventData?.projectEvents?.eventLogEntries;
+
   return (
-    <>
-      <div>This is the event log page for {identifier}</div>
-      <div> {JSON.stringify(projectEventData)}</div>
-      <div> {JSON.stringify(repoEventData)}</div>
-    </>
+    <div>
+      <div data-cy="event-log">
+        {((eventData as logEntry[]) || []).map(
```

Comment:
We shouldn't need this type aliasing.

Reply:
it's needed in order to be able to call .map on it